packages/neo-one-server-plugin-project/src/initCommand.ts

```diff
@@ -34,8 +34,16 @@ export const initCommand = ({ cli }: InteractiveCLIArgs) =>
   cli.vorpal.command('init', 'Initialize neo-one in the current directory.').action(async () => {
     const cwd = process.cwd();
     const projectConfig = await loadProjectConfig(cwd);
-    const tsconfigPath = path.resolve(projectConfig.paths.contracts, 'tsconfig.json');
+    const relativeConfig = {
```

Comment:
let's just spread the remaining properties so that we don't forget to update this if we add more to the `ProjectConfig` later:

```
const relativeConfig = {
  ...projectConfig,
  paths: {
    ...projectConfig.paths,
    contracts: path.relative(process.cwd(), projectConfig.paths.contracts),
    generated: path.relative(process.cwd(), projectConfig.paths.generated),    
  },
};
```

Reply:
When I did the spread, I had my order backwards, Thnx.